src/openmc_mcnp_adapter/openmc_conversion.py

```diff
@@ -258,9 +258,9 @@ def flip_sense(surf):
             displacement, rotation = data['tr'][tr_num]
             with warnings.catch_warnings():
                 warnings.simplefilter("ignore", openmc.IDWarning)
-                surf = surf.translate(displacement, inplace=True)
                 if rotation is not None:
                     surf = surf.rotate(rotation, inplace=True)
```

Comment:
actually instead of changing the order of parts of the code, you can just add the `pivot` kwarg to the `rotate` method like so: `surf = surf.rotate(rotation, pivot=displacement, inplace=True)` You should be able to do this on line 365 as well, but in that part of the code `vector` is used instead of `displacement`